assets/app/view/spreadsheet.rb

```diff
@@ -96,10 +97,20 @@ def render_title
     end
 
     def render_corporations
-      @game.corporations.map { |c| render_corporation(c) }
+      current_round = [@game.turn]
+      current_round += [@game.round.round_num] if @game.round.name == 'Operating Round'
+
+      floated_corporations = @game.round.entities
+      @game.corporations.map do |c|
+        operating_order = -1
```

Comment:
this seems useless

Reply:
Needed some sort of a marker to differentiate between floated and unfloated corps